### PR TITLE
Add band name to the roles view

### PR DESCRIPTION
**Pull request by bannister-samuel**
RolesDao update to pull band_name into the jobRoles object
Add band_name to JobRole json object to be returned

### CODE REVIEW

**Review by shaunganley — CHANGES_REQUESTED** [review]
Unit and integrations tests updated please

**Review by TomaszLaski — COMMENTED** [review]
All seems to be ok

File: src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java [hunk 1]
```diff
@@ -21,13 +25,26 @@ public class WebServiceIntegrationTest {
...
+    void getJobRoles_shouldReturnListOfJobRoles_withValidBandName() {
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode response = APP.client().target("http://localhost:8080/api/job-roles")
+                .request()
+                .get(JsonNode.class);
+        List<JobRole> JobRoleList = mapper.convertValue(
+                response, new TypeReference<List<JobRole>>(){}
+        );
+
+        Assertions.assertEquals(JobRoleList.get(0).getBandName(), "Executive");
```
Comment (shaunganley) [inline src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java]: can you check this as part of an existing test to reduce the number of http calls required

**Review by shaunganley — CHANGES_REQUESTED** [review]

File: src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java (hunk 1 above)
Comment (bannister-samuel) [reply src/test/java/com/kainos/ea/integration/WebServiceIntegrationTest.java]: done

**Review by shaunganley — APPROVED** [review]

**Review by cdrainkainos — APPROVED** [review]
all good

**Pull request merged**